src/components/json/worldData.js

```diff
@@ -335,8 +336,8 @@ const data = [
         "portfolio": "http://aravindh020.github.io/"
 },
 {
-	"longitude": 76.958896,
-	"latitude": 11.000897,
+	"longitude": 77.620540,
+	"latitude": 13.079790,
```

Comment:
Any specific reason behind changing Naveen's geolocations?

Reply:
Really sorry, didn't notice that. Made another request.